Create service_eval.md

anees1203 opened the pull request:
service_eval.md file

anees1203 committed:
Create service_eval.md

service_eval.md file

anees1203 committed:
Merge branch 'oss-slu:main' into anees1203-patch-1

anees1203 committed:
Update service_eval.md

Servant as Leader Evaluation

anees1203 committed:
Update service_eval.md

anees1203 commented:
Servant as Leader Evaluation on service_eval.md file (WWF)

kungfuchicken reviewed (changes requested):
Please, cite any sources used to generate this evaluation. This does not need to be an academic citation. Links to information on the web is sufficient. Just ensure you cite specific pages, and not just the organization's home page.

This evaluation does not connect the Theory of Change framework to specific elements of the analysis. It is mentioned, and there seem to be some implicit elements. Please make the implicit elements that relate to ToC explicit.

The Servant Leader framework was not requested.

anees1203 committed:
Update service_eval.md

I apologize for the oversight in not explicitly connecting the Theory of Change (ToC) framework to the evaluation of the World Wildlife Fund (WWF).

anees1203 commented:
I apologize for the oversight in not explicitly connecting the Theory of Change (ToC) framework to the evaluation of the World Wildlife Fund (WWF). I have made changes! Please take a look at it.
